src/pages/index.astro

```diff
@@ -48,3 +48,10 @@ const defaultEntries = entries.map((entry) => entry.default);
     </div>
   </body>
 </html>
+
```

Comment:
```suggestion

```

Extra line here kub :eyes:

Reply:
Should we do this as git commit hooks?

Reply:
Not sure if we integrate tools such as husky will impact any dx on the new comer side, but will take it to consideration kub :pray: